modules/installation/pages/proxy-setup.adoc

```diff
@@ -209,22 +225,43 @@ If you do not use a reactivation key, the replacement proxy will become a "new"
 ====
 
 
-[[proxy.migration3.replace]]
-.Procedure: Replacing a Proxy and Keeping the Clients Registered
-. Before starting the actual migration procedure, save the data from the old proxy, if needed.
-    Consider copying important or manually created data to a central place that can also be accessed by the new proxy.
-. Shut down the proxy.
+.Procedure: Replacing a Traditional Proxy and Keeping the Clients Registered
+. Before starting the migration, save the data from the old proxy, if needed.
+  Consider copying important or custom data to a central place that can also be accessed by the new proxy.
+. Shut down the old proxy.
 . Install a new {productname} Proxy.
-    For installation instructions, see xref:install-proxy-unified.adoc[Proxy Installation].
+  For installation instructions, see xref:install-proxy-unified.adoc[Proxy Installation].
 . In the {productname} {webui}, select the newly installed {productname} Proxy, and delete it from the systems list.
-. In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.
+. In the {webui}, create a reactivation key for the old proxy system.
+  On the [guimenu]``System Details`` tab of the old proxy click [guimenu]``Reactivation``.
   Click [guimenu]``Generate New Key``, and make a note of the new key.
-  For more information about reactivation keys, see xref:reference:systems/system-details/sd-reactivation.adoc[Reactivation Keys].
-. Register the proxy with a bootstrap script as described in xref:installation:proxy-registration.adoc[].
-  In the bootstrap script set the reactivation key as [systemitem]``ACTIVATION_KEYS``.
-. Re-run the configure-proxy.sh as described in xref:installation:uyuni-proxy-setup.adoc#uyuni-proxy-setup-confproxy[].
-. OPTIONAL: After the installation of the new proxy, you might also need to:
-+
+. Register the new proxy with a bootstrap script as described in xref:installation:proxy-registration.adoc[].
+  In the bootstrap script, set the reactivation key with the [systemitem]``REACTIVATION_KEY`` parameter.
+. Restore the proxy from the backup you made earlier.
```

Comment:
Maybe, we could rephrase this.  See the backport source ;)

Reply:
Let's not do that in a backport. Let's merge this as-is, and then do a new PR to make changes.